app/controllers/decidim/accountability/admin/import_results_controller.rb

```diff
@@ -13,7 +13,7 @@ def create
           @csv_file = params[:csv_file]
           redirect_to(new_import_path) && return if @csv_file.blank?
 
-          Decidim::Accountability::Admin::ImportResultsCSVJob.perform_later(current_user, current_feature, @csv_file.read)
+          Decidim::Accountability::Admin::ImportResultsCSVJob.perform_later(current_user, current_feature, @csv_file.read.force_encoding('utf-8').encode('utf-8'))
```

Comment:
Another way to fix this would be to read the CSV *here* and send the JSON-friendly data as a hash. Sounds a bit more robust, but either way, this works.